src/components/UI/ErrorFallback.tsx

```diff
@@ -16,7 +16,10 @@ interface IErrorFallbackProps {
  * This component will return the error passed to it,
  * if it exists, or the children, if it does not
  */
-function ErrorFallback({ error, children }: IErrorFallbackProps) {
+function ErrorFallback({
```

Comment:
Not a blocker from me, but why don't you like using the function expression? 😬

Reply:
I'm ok with both, I tend to use function declarations in general because of the hoisting and because I found them more readable at the top level. That's it :-)

Reply:
I don't think we'll have any problems with using it this way, but syntactically, it's no longer a react component, it's just a function that returns a react element 🤔... Guess we'll find out in the future if something breaks, it's not a hard fix

Reply:
A function that returns a react element it is a component, a functional component, isn't it? That'a the whole move of hooks AFAIU.

And it is basically the exact same than a function expression that returns a react element, with some little differences.

Reply:
Functionality-wise, expressions vs function components are the same.

What I meant that for TypeScript, `React.FC`, which is the function component type, is not the same as the type of the function declared here. So any function/generic type/etc that takes a react component (being it `FC` or `ClassComponent`), won't work with the function above.

That's why I said that we should see in the future if it will throw any errors

Reply:
It looks like TypeScript doesn't support this: https://github.com/microsoft/TypeScript/issues/22063

So we have to use function expressions.